### PR TITLE
Moving Tutorial from README to Jupiter notebook. 

### PR DESCRIPTION
**Pull request by nrich20**
I have removed the tutorial from the README file and have put a small note in its place with a link to the Jupyter Notebook tutorial. The link will not currently work since its set to access the master branch, but the Jupyter tutorial doesn't exist in that branch until this merge.  Also, tutorial.ipynb can open and run in colab directly from github.

### CODE REVIEW

File: Jupyter_notebook/tutorial.ipynb [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "PIA2IaqUOeOA"
+      },
+      "source": [
+        "# Welcome to the NoisePy Colab Tutorial!\n",
+        "\n",
+        "This tutorial will walk you through the basic steps of using NoisePy to compute ambient noise cross correlation functions.\n",
```
Comment (carlosgjs) [inline Jupyter_notebook/tutorial.ipynb:11]: Please take a look at the latest version of the [colab notebook](https://colab.research.google.com/drive/1ir-p05dW34cj4biEZYqa7QXgXpjqv3fW#scrollTo=rR860zaZPZGH). The installation is now done through:
```
!pip install noisepy-seis
```
So there's no need to clone the repo, install requirements or cd to `src`.

File: Jupyter_notebook/tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
...OrZjNz585FbGyss6vBYDCSEUyTx2AwGIzkzMOHD51dBZvwCCEvPj4eZ86ckeyT8vLyQnh4uENcqtuDuLg43LlzB0OHDkXXrl2dXR0Gg5HMYCbJDAaDwUjOjBw5Erdu3XJbyxb3dhvz/zx//hyJiYnInDmz5HjmzJlx/fp1q/RxcXGSBxYTE2P3Ourl5MmTTnW0wmAwGAwGg8FgJFfWr1+P9evXo3r16ti3b5+zq6Mbj9Dk6WXcuHFIly6d8C9HjhzOrpIVRYsWxV9//eXsajAYDAaDwWAwGMmOPHnyYPz48Rg1apSzq2IIj9DkZcyYEd7e3oiMjJQcj4yMRHBwsFX6wYMHo2/fvsLvmJgYlxP0goKC8OWXXyIiIgKnTp1CgwYNnF0lBoPBYDAYDAbD48mbNy9u377t7GrYhEdo8nx9fREaGoq9e/cKx5KSkrB3715iUE4/Pz+kTZtW8s9VyZw5M3Lnzu3sajAYDAaDwWAwGMkCT/Bq7xFCHgD07dsXf/zxB5YsWYJr166hW7duePv2Ldq3b+/sqtlMqlSpnF0FBsMlKF68uLOrwGAwkinp0qVzdhUYDIaD8PJyfxHJI8w1AaB58+Z49uwZhg0bhoiICJQqVQo7d+60csbijvBCXlhYGD58+IDTp087uUYMhnOYMWMGqlat6uxqMBiMZIifn5+zq8BgMBjUeIyQBwA9e/ZEz549nV0N00mdOjUAIFeuXLhz547dyvH29kZiYqLd8mcwbCVFihTOrgLDA2FjH4MGNv4wGAx3wv11kcmAlClTYvPmzZg7dy4sFovdyvH19bVb3gyGGfj4eNS6lCp58+Z1dhWSDZ5glsMwF5LTNvaN/Ei9evWcXQUGw+7Yc77tKNiXzU1o0KAB0qRJY9fJCPuASZk8ebKzq2AzGTJkcHYVTMXZK+l37951WFnXrl1zWFnJHSbkMeQEBAQgY8aMkmPOHn9cBU9wSMGw5vPPP3d2FZzOrFmz8OzZM2dXwzTYl83NsOfKgr+/v93ydkcCAwOJx3fv3o3Ro0cjf/78jq2QAa5cueLsKpiKszV5jpzcsAml43B2v2K4Jq4w2cuZMyf69++veD4kJMTUecHcuXPRqlUr0/JjMNwJi8UifHuZJo/hcOy54sw2lUtR0oKFh4fjp59+QqFChRxcI/1oCQquHD6EhLNXkB1RfsGCBTFlyhS7l8
```
Comment (carlosgjs) [inline Jupyter_notebook/tutorial.ipynb:278]: The last time I was playing with it, this step wasn't working. Can you do a little digging to see what may be going on?

**Review by carlosgjs — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+After cloning the repo and creating a virtual environment with either **pip** o **conda**:
+
+Do an editable installation to get the dependencies (from the project root):
+```sh
+$ pip install -e ".[dev]"
+Do an editable installation to get the dependencies (from the project root):
+```sh
+$ pip install -e ".[dev]"
```
Comment (carlosgjs) [inline CONTRIBUTING.md:10]:
```suggestion
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+Install the `pre-commit` hook:
+```sh
+$ pre-commit install
+
+Install the `pre-commit` hook:
+```sh
+$ pre-commit install
```
Comment (carlosgjs) [inline CONTRIBUTING.md:19]:
```suggestion
```

File: Jupyter_notebook/tutorial.ipynb [hunk 5]
```diff
@@ -30,41 +17,93 @@
...
+        "id": "rR860zaZPZGH",
+        "outputId": "beed533c-e81e-429a-ce2f-d36dc154b248"
       },
       "outputs": [
         {
           "name": "stdout",
           "output_type": "stream",
           "text": [
-            "/content/NoisePy/src\n"
+            "Collecting noisepy-seis\n",
```
Comment (carlosgjs) [inline Jupyter_notebook/tutorial.ipynb:39]: Can you clear all the outputs before committing the notebook?

File: README.md [hunk 6]
```diff
@@ -9,6 +9,7 @@ Detailed documentation can be found at https://noisepy.readthedocs.io/en/latest/
 [![Build Status](https://travis-ci.com/chengxinjiang/NoisePy.svg?branch=master)](https://travis-ci.com/github/chengxinjiang/NoisePy)
 [![Codecov](https://codecov.io/gh/chengxinjiang/NoisePy/branch/master/graph/badge.svg)](https://codecov.io/gh/chengxinjiang/NoisePy)
 
+<img src="https://raw.githubusercontent.com/mdenolle/NoisePy/master/docs/figures/logo.png" width="800" height="400">
```
Comment (carlosgjs) [inline README.md:12]: duplicated?
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -17,9 +18,46 @@ Jiang, C. and Denolle, M. "NoisePy: a new high-performance python tool for seism
 
 ## Major updates include
 * adding options for several stacking methods such as nth-root, robust-stacking, auto-covariance and selective. A script is added to the folder of application_modules to cross-compare the effects of different stacking method (note that `substack` parameter in S2 has to be `True` in order to use it)
+* adding options for several stacking methods such as nth-root, robust-stacking, auto-covariance and selective. A script is added to the folder of application_modules to cross-compare the effects of different stacking method (note that `substack` parameter in S2 has to be `True` in order to use it)
```
Comment (carlosgjs) [inline README.md:21]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -17,9 +18,46 @@ Jiang, C. and Denolle, M. "NoisePy: a new high-performance python tool for seism
...
 
+# Installation
+The nature of NoisePy being composed of python scripts allows flexible package installation, which is essentially to build dependent libraries the scripts and related functions live upon. We recommend using [conda](https://docs.conda.io/en/latest/) or [pip](https://pypi.org/project/pip/) to install the library due to their convenience. Below are command lines we have tested to create a python environment to run NoisePy. Note that the test is performed on `macOS Mojave (10.14.5)`, so it could be slightly different for other OS.
+
+
+### Note the order of the command lines below matters ###
+
+# With Conda:
+```bash
+$ conda create -n noisepy python=3.8 pip
```
Comment (carlosgjs) [inline README.md]: Also several duplicated lines in this section, probably from a bad merge

File: test/test_routines/test_stretching.py [hunk 9]
```diff
@@ -0,0 +1,65 @@
+import time
```
Comment (carlosgjs) [inline test/test_routines/test_stretching.py:1]: this file was moved to `unittests/.`, can you remove it from here?

**Review by carlosgjs — CHANGES_REQUESTED** [review]
Getting there! Looks like there are some artifacts from merge conflicts.

**Review by niyiyu — APPROVED** [review]
Thanks Nick!

**Review by carlosgjs — APPROVED** [review]

**Pull request merged**